libs/entitlement/bridge/src/main/java/org/elasticsearch/entitlement/bridge/EntitlementChecker.java

```diff
@@ -560,6 +561,27 @@ public interface EntitlementChecker {
 
     void check$java_nio_file_Files$$setOwner(Class<?> callerClass, Path path, UserPrincipal principal);
 
+    ////////////////////
+    //
+    // Thread management
```

Comment:
This doesn't look like the right place in the file. These are in the middle of the file entitlement methods.

Reply:
Oh probably from merge conflicts.

Reply:
Let's not do that here.